MIPS/mip-0005-mip-upgrade.md

```diff
@@ -0,0 +1,315 @@
+- The proposal is original or new.
+- The proposal is clearly articulated.
+- The proposal uses the MIP Template correctly.
+- The proposal is not missing any required parts.
+
+If the pull request is rejected, the MIP Facilitator will communicate the decision to the MIP Author explaining the reasons for the rejection and a list of changes that could be made for the proposal to be accepted, so that the MIP Author can make these changes.
+
+If the pull request is accepted, the MIP Facilitator:
+- Approves the MIP and assigns it a formal MIP number.
+- Changes the status of the MIP.
+- Merges in the PR.
+- Communicates the decision to the MIP author.
+
+**Review Phase - Part 1: Risk Assessment**
+We propose to create a pool of experts to review MIPs across the 3 categories. For each MIP, the Author reaches out to the pool of experts for the relevant category and sends a Request for Comments. To maintain independence, the Author does not choose who the Reviewers are. This MIP Review Group has a period to review the MIP and provide their feedback. The MIP Author can incorporate the feedback into the MIP until a final draft is completed. If the MIP Author does not incorporate the feedback, the MIP process still continues.
+
+When the MIP Author considers the draft ready for the final review, editing stops for at least 1 week to allow MIP Reviewers to write individually a Risk Assessment that is incorporated into the designated section of the MIP template. 
+
+The MIP Facilitator must check that every MIP has enough reviews to complete the Review phase (at least 3 if there is agreement, 5 otherwise). The MIP Facilitator can also coordinate communications and collaborate with the MIP Author to schedule any necessary meetings, and to define a roadmap with target dates for each milestone in the MIP process. 
+
+**Review Phase - Part 2: Community Conversation with Mass Deliberation**
+The MIP Facilitator sets a review period for the community to review the MIP, including the Risk Assessment, and schedules an open call where the MIP Author and MIP Review Group can present the proposal in detail. During this call, they will discuss the risks, benefits, and tradeoffs, address questions from the community, and provide instructions for on-chain voting, including the voting dates.
+
+After this meeting, the community will have some time again to reconsider the proposal before the voting takes place.
+
+https://drive.google.com/file/d/1RWpwc08Oj43zE25oKRdSmh-Gud_8RNfl/view?usp=drive_link
+
+### Guidelines for Implementation
+
+Every MIP should include an implementation plan that explicitly states what exactly needs to be done to implement the proposed changes, who is responsible for this implementation (in agreement with the Dev Team) and what are the target dates for each milestone. 
+
+MIP Authors are responsible for writing the implementation plan, collaborating and establishing connections with the Dev Team and other stakeholders during the implementation. 
```

Comment:
Agree with other commenters that this may be a separate role.

Reply:
Noted, I will give further thought to how we can more clearly define the separation between the MIP Author and the implementation team.